services/messagebus.md

```diff
@@ -3,20 +3,18 @@
     identifier: 0x17feef9b
     
 A lightweight event bus that allows to broadcast
-short messages.
+short messages, similar to the Scratch message APIs.
```

Comment:
Is the service just going to raise the event after it gets a send?

Reply:
so basically the idea is that when the device receives a JACDAC event; it raises it on the CODAL event bus side.

Reply:
so there is no generally-agreed-upon meaning to these events? If it's for CODAL, it should be called CODAL Message Bus; but I think we also need a clear usage scenario.